[2023-06-09T14:53:17.496+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_into_table scheduled__2023-06-06T00:00:00+00:00 [queued]>
[2023-06-09T14:53:17.502+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_into_table scheduled__2023-06-06T00:00:00+00:00 [queued]>
[2023-06-09T14:53:17.502+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 7
[2023-06-09T14:53:17.509+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): delete_into_table> on 2023-06-06 00:00:00+00:00
[2023-06-09T14:53:17.513+0000] {standard_task_runner.py:57} INFO - Started process 1864 to run task
[2023-06-09T14:53:17.518+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_into_table', 'scheduled__2023-06-06T00:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp3uczhfav']
[2023-06-09T14:53:17.520+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask delete_into_table
[2023-06-09T14:53:17.562+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_into_table scheduled__2023-06-06T00:00:00+00:00 [running]> on host 94560a00d375
[2023-06-09T14:53:17.620+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='znawawi' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-06-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-06T00:00:00+00:00'
[2023-06-09T14:53:17.621+0000] {sql.py:262} INFO - Executing: 
            delete from dag_runs where dt = '2023-06-06' and dag_id = 'dag_with_postgres_operator_v04'
        
[2023-06-09T14:53:17.629+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-09T14:53:17.667+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-09T14:53:17.676+0000] {sql.py:375} INFO - Running statement: 
            delete from dag_runs where dt = '2023-06-06' and dag_id = 'dag_with_postgres_operator_v04'
        , parameters: None
[2023-06-09T14:53:17.679+0000] {sql.py:384} INFO - Rows affected: 0
[2023-06-09T14:53:17.691+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_into_table, execution_date=20230606T000000, start_date=20230609T145317, end_date=20230609T145317
[2023-06-09T14:53:17.727+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-09T14:53:17.747+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
